dynwalls/__main__.py

```diff
@@ -65,30 +63,30 @@ def use(self, arguments):
                 print("Warning: Ambigous time specifiation found. Might skip some images.")
             prev = cur
 
-        # TODO: check if all image indizes are available in the heic file
+        # TODO: check if all image indices are available in the heic file
 
         # clean up old images
         if os.path.isdir(WP_DIR):
             for f in os.listdir(WP_DIR):
                 if re.match(f"^{PREFIX}-[\\d]+{EXTENSION}$", f):
                     os.remove(f"{WP_DIR}/{f}")
 
-        heic.extract_images(arguments.heicfile,
-                            outputdir=WP_DIR,
-                            filename_prefix=PREFIX,
-                            extension=EXTENSION)
+        heic.extract_images(
+            arguments.heicfile,
+            outputdir=WP_DIR,
+            filename_prefix=PREFIX,
+            extension=EXTENSION,
+        )
 
         config.dyn_config = c
 
         # create units
         timelist = self.get_timelist()
         systemd.setup_units(timelist)
         # TODO: only update if enabled
-        self.update({})
```

Comment:
As the `arguments` parameter was not used in the `update` method line 101, I removed it from here and from line 101. I did the same for the methods that was not using arguments.

Reply:
As the `arguments` parameter was not used but how the script works requires to have two arguments on the `update` method, I set a default value on line 89 so we can call `update` without arguments. Same apply to `enable` and `disable` methods.

Reply:
The methods of the `dynwalls` class currently mirror the command line subcommands (`setcmd`, `enable`, `use`, `disable`, `update`). The `act` method is using meta-programming by dynamically calling a method on itself with the name of the `actionname` variable. This makes the code more compact and you don't need long switch-case expressions (`if... elif...`) in python. There is also some meta-programming in the `config.py` (overriding `getattr` and `setattr`) and `args.py` file. This can confuse linters.

Great idea to give them default values :+1: